src/Report/main.tex

```diff
@@ -224,16 +257,104 @@
+            
+                        
             \begin{figure}
-                \includegraphics[width=\textwidth]{assets/results/definition/definition.pdf}
-                \caption{Korrektheit der eigenen Definition nach Selbsteinschätzung geordnet.}
-            \end{figure}
-        
-            \begin{figure}
-                \includegraphics[width=\textwidth]{assets/results/openSourceJudging/openSourceJudging}
-                \caption{Attribut Open Source  |  unbeeinflusste Nutzereinschätzung}
+                \includegraphics[width=\textwidth]{assets/results/openSourceJudging/openSourceJudgingDetailedOSSOnly2.pdf}
+                \caption{Abweichung des Wissens über \gls{oss} von $50\%$.}
+                \label{figure:knowledge_difference}
             \end{figure}
-            Das Wissen über Open Source Software bei Endanwendern ...
+            
+            
+            \subsubsection{Aufbau des Diagramms}
+                \paragraph{Kategorien}
+                    Für die Auswertung wurden die Softwares wie auch schon bei der Umfrage in die Kategorien Browser, Betriebssysteme (OS), Web, Desktop und Spiele gruppiert.
+                    
+                \paragraph{Werte}
+                    Zunächst ist zu betrachten, wie das Ergebnis ausfallen würde, wenn alle Befragten raten würden. In diesem Fall wäre der Erwartungswert $50\%$ der Befragten, da es sich um eine Ja-Nein-Frage, also eine Frage mit nur zwei Möglichkeiten, handelt.
+                    
+                    Um also die Ausgangsfrage bewerten zu können ist es notwendig sich die Differenz zwischen $50\%$ und dem Ergebnis der Umfrage zu betrachten. Dabei kann es zu drei möglichen Fällen kommen:\\
+            
+                    \begin{tabularx}{\textwidth}{rX}
+                        \textbf{Ergebnis} & \textbf{Auswertung} \\\hline
+                        $= 50\%$ & Die Befragten haben haben das selbe Ergebnis erreicht, wie beim Raten. Das bedeutet, dass im Durchschnitt kein Wissen über \gls{oss} vorhanden ist.\\
+                        $> 50\%$ & Ein größerer Teil der Befragten konnten die Frage richtig beantworten. Das bedeutet, dass im Durchschnitt richtiges Wissen über \gls{oss} vorhanden ist. Im Extremfall 100\% konnte jeder der Befragten die Frage richtig beantworten.\\
+                        $< 50\%$ & Ein größter Teil der Befragten hat die Frage falsch beantwortet. Das bedeutet, dass im Durchschnitt falsches Wissen über \gls{oss} vorhanden ist. Im Extremfall 0\% hat keiner der Befragten die Frage richtig beantworten.  
+                    \end{tabularx}
+            
+                \paragraph{Auswahl der Datenpunkte}
+                    Bei der Auswertung wurden nur die Antworten von Nutzern gewertet, die auch angaben die Software zu verwendet um sicher zu stellen, dass den Nutzern die Software bekannt ist. Andernfalls würde das Ergebnis verfälscht werden.
```

Comment:
`gewertet` durch `in Betracht gezogen` ersetzen um Wortwdhl mit `Auswertung` zu umgehen.